nixos/modules/services/scheduling/marathon.nix

```diff
@@ -69,21 +80,21 @@ in {
 
   config = mkIf cfg.enable {
     systemd.services.marathon = {
-      description = "Marathon Service";
+      description = mkDefault "Marathon Service";
       environment = cfg.environment;
       wantedBy = [ "multi-user.target" ];
       after = [ "network-interfaces.target" "zookeeper.service" "mesos-master.service" "mesos-slave.service" ];
 
       serviceConfig = {
-        ExecStart = "${pkgs.marathon}/bin/marathon --master ${cfg.master} --zk zk://${concatStringsSep "," cfg.zookeeperHosts}/marathon --http_port ${toString cfg.httpPort} ${concatStringsSep " " cfg.extraCmdLineOptions}";
-        User = "marathon";
-        Restart = "always";
-        RestartSec = "2";
+        ExecStart = mkDefault "${pkgs.marathon}/bin/marathon --master ${cfg.master} --zk zk://${concatStringsSep "," cfg.zookeeperHosts}/marathon --http_port ${toString cfg.httpPort} ${concatStringsSep " " cfg.extraCmdLineOptions}";
```

Comment:
You are right. I thought it works anywhere. I should have tested first. I have removed the `mkDefault`s.